tools/Cargo.lock

```diff
@@ -1291,8 +1100,9 @@ dependencies = [
  "tempfile",
  "tinytemplate",
  "tokio",
+ "tokio-stream",
  "toml",
- "tough",
+ "tough 0.11.0",
```

Comment:
It's unfortunate that we need tough 0.10 as well because we link to update_metadata from sources/.  Could that potentially cause a problem?  I think I trust the compiler here, in that we know we're not intermingling different versions of tough types, but as we know from tokio runtimes, it's possible for there to be failure modes that only show up with a bang.  Do you feel OK here?

Reply:
I took a look at `update_metadata` and I don't think `pubsys` is touching `tough` (or `tokio`) via `update_metadata`. It's kind of a weird crate.

I think the only use of `tough` in `update_metadata` is this function:

```rust
pub fn load_manifest(repository: &tough::Repository) -> Result<Manifest> {
    let target = "manifest.json";
    serde_json::from_reader(
        repository
            .read_target(target)
            .context(error::ManifestLoad)?
            .context(error::ManifestNotFound)?,
    )
    .context(error::ManifestParse)
}
```

That should probably be pasted into `updog` and `migrator` to eliminate the `tough` dependency from `update_metadata`. (I'm the one that must have created this dependency when working on signed migrations).